casepro/msgs/tasks.py

```diff
@@ -1,9 +1,23 @@
 from __future__ import absolute_import, unicode_literals
 
+import csv
+import traceback
 from celery import shared_task
+from celery.task import task
 from celery.utils.log import get_task_logger
+from django.db import transaction
+from django.utils import timezone
+from django.conf import settings
 from dash.orgs.tasks import org_task
 from datetime import timedelta
+from smartmin.csv_imports.models import ImportTask
+from .models import FAQ, Language, Label
+
+# python2 and python3 support
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
I've not written a lot of Python 3 compatible code so might be wrong, but doesn't `io.StringIO` exist since Python 2.6?